ocs_ci/utility/utils.py

```diff
@@ -408,6 +410,21 @@ def run_cmd(cmd, secrets=None, **kwargs):
     return mask_secrets(r.stdout.decode(), secrets)
 
 
+def run_mcg_cmd(cmd, namespace='openshift-storage'):
```

Comment:
can you enhance the run_cmd to handle noobaa cmds too, thanks

Reply:
Handle in what sense? :)

Reply:
yeah if you can enhance `run_cmd` function to handle noobaa as well.

Reply:
I disagree here, the run_cmd is the general one function to run whatever command and shouldn't be bound with nooba or whatever else.  This function is just wrapper which will automatically pass the nooba command at the start of cmd + namespace.

So I would keep this in separate wrapper function like it is.

Reply:
I agree with Petr. I think adding complexity to `run_cmd` might cause unexpected complications in the future, and it means I'll have to add some sort of an MCG parameter to each and every function call (in which case it'd still be safer and easier to call `run_cmd('noobaa...')`

Reply:
The only difference right now I see is - it appends noobaa, I dont have a strong preference here *as long as it does the same and we dont duplicate code*, if its part of run_cmd, the duplication can never happen.

Reply:
this function is wrapping run_cmd, it adds noobaa to the cmd + it also add namespace which is related to that command. So for usage it will be easier to use this one wrapper function in my opinion.

And probably later on add some more functionality/parameters here.  I don't like the idea to put any other command specific stuff to general run_cmd function as it is suppose to run whatever command. So if Vasu is not strongly against this I would resolve this conversation if @vasukulkarni  is ok with this.

Reply:
Resolving this, I can wait and watch how much noobaa cli differs.